.github/workflows/build_deploy.yml

```diff
@@ -44,12 +91,15 @@ jobs:
       - name: Run stage build
         if: github.ref == 'refs/heads/stage'
         env:
-          NODE_OPTIONS: '--openssl-legacy-provider'
+          NODE_OPTIONS: "--openssl-legacy-provider"
```

Comment:
remove this and other unused envs (all except new ones?)
do it only for stage build for now